Makefile

```diff
@@ -105,6 +105,10 @@ scala-artifact = \
 .PHONY: all
 all: docker-images
 
+.PHONY: env
+env:
```

Comment:
We should call it `docker-env` or `docker-compose-env` to keep the naming consistent.